### PR TITLE
Add parse windows volume

### PR DESCRIPTION
**Pull request by aiyijing**
Add parse windows volume

Close https://github.com/kubernetes/kompose/issues/1368

Signed-off-by: aiyijing <aiyijing@live.com>

### CODE REVIEW

File: pkg/transformer/utils.go [hunk 1]
```diff
@@ -112,6 +120,77 @@ func ParseVolume(volume string) (name, host, container, mode string, err error)
 	return
 }
 
+func parseWindowsVolume(volume string) (name, host, container, mode string, err error) {
```
Comment (hangyan) [inline pkg/transformer/utils.go:123]: add example volume string in the comment so others can easily understand this function

Comment (aiyijing) [reply pkg/transformer/utils.go:131]: done

**Comment by hangyan** [comment]
Code lgtm to me now, but a compose yaml test would be better

File: script/test/cmd/tests_new.sh [hunk 2]
```diff
@@ -106,3 +106,9 @@ convert::expect_warning "$cmd" "Push image registry 'whatever' is specified but
 #TEST the kompose.volume.storage-class-name label
 convert::check_artifacts_generated "kompose -f $KOMPOSE_ROOT/script/test/fixtures/storage-class-name/docker-compose.yml convert -o $TEMP_DIR/output-k8s.json -j" "$TEMP_DIR/output-k8s.json"
 convert::check_artifacts_generated "kompose --provider=openshift -f $KOMPOSE_ROOT/script/test/fixtures/storage-class-name/docker-compose.yml convert -o $TEMP_DIR/output-os.json -j" "$TEMP_DIR/output-os.json"
+
+# TEST the windows volume
+# windows host path to windows container
```
Comment (hangyan) [inline script/test/cmd/tests_new.sh:111]: add openshift test too.

Comment (aiyijing) [reply script/test/cmd/tests_new.sh:111]: done

**Comment by hangyan** [comment]
@aiyijing Thanks for your contribution

**Pull request merged**